src/components/MapDisplay.css

```diff
@@ -0,0 +1,4 @@
+#map {
```

Comment:
any specific reason for  having styles with `ID`?

Reply:
Mapbox API selects div element using it's ID.